packages/core/src/anchor/__tests__/ethereum/ethereum-anchor-validator.test.ts

```diff
@@ -0,0 +1,92 @@
+import { jest } from '@jest/globals'
+import { LoggerProvider, AnchorValidator, AnchorProof } from '@ceramicnetwork/common'
+import { CID } from 'multiformats'
+
+const TEST_CHAIN_ID = '1337'
+const ANCHOR_PROOF: AnchorProof = {
+  chainId: `eip155:${TEST_CHAIN_ID}`,
+  blockNumber: 3,
+  blockTimestamp: 1586784008,
+  txHash: CID.parse('bagjqcgza7mvdlzewbfbq35peso2atjydg3ekalew5vmze7w2a5cbhmav4rmq'),
+  root: CID.parse('bafyreic5p7grucmzx363ayxgoywb6d4qf5zjxgbqjixpkokbf5jtmdj5ni'),
+  version: 1,
+}
+const TEST_TRANSACTION = {
+  data: '0x7d3b0ca2000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000250f017112205d7fcd1a0999befdb062e6762c1f0f902f729b98304a2ef539412f53360d3d6a000000000000000000000000000000000000000000000000000000',
+  blockNumber: ANCHOR_PROOF.blockNumber,
+  blockHash: '0x752fcd3593c140db9f206b421c47d875e0f92e425983f8c72ab04c0e969b072a',
+  to: '0xD3f84Cf6Be3DD0EB16dC89c972f7a27B441A39f2',
+}
+
+const MockJsonRpcProvider = {
+  getTransaction: null,
+  getNetwork: null,
+  getBlock: null,
+
+  reset() {
+    this.getTransaction = jest.fn(() => {
+      return Promise.resolve(TEST_TRANSACTION)
```

Comment:
would be good if this checked that the txHash requested matches what we expect (the txHash from the ANCHOR_PROOF)